src/alchemlyb/tests/test_workflow_ABFE.py

```diff
@@ -1,9 +1,11 @@
 import os
 
 import numpy as np
+import pandas as pd
 import pytest
 from alchemtest.amber import load_bace_example
 from alchemtest.gmx import load_ABFE
+from joblib import parallel_config
```

Comment:
Maybe just `import joblib` so that it's clearer below what comes from joblib? In this way any un-qualified functions and classes are alchemlyb and everything else is external.